lib/client-api/src/story_store.js

```diff
@@ -126,8 +95,8 @@ export default class StoryStore extends EventEmitter {
     );
   }
 
-  setSelection = data => {
-    this._selection = data;
+  setSelection = ({ storyId }) => {
+    this._selection = { storyId };
```

Comment:
If the only datapoint is the storyId, Why make it an object?

Reply:
I think @shilman has secret plans and clever tricks he is not telling you about in this PR

Reply:
(The answer is later there'll be `docs: boolean` as a second datapoint)